randovania/games/planets_zebeth/logic_database/Brinstar.txt

```diff
@@ -276,29 +271,22 @@ Extra - map_name: Brinstar/x312_y30
 > Door to Corridor 3; Heals? False
   * Layers: default
   * Normal Door to Corridor 3/Door to Corridor-connecting room
-  > Bridge
+  > Door to Varia Suit corridor
+      Hi-Jump Boots
```

Comment:
i wonder if you can do some ice beam abuse here

Reply:
Confirmed doable. It seems very difficult to pull off without bombs and could be a potential soft lock if the enemy gets caught in the respawning blocks.

Reply:
Found a better setup, only need Ice Beam <https://youtu.be/y4qCIB5QAL8>, still could get stuck if you morph.